scripts/compare.py

```diff
@@ -125,13 +126,23 @@ def compare(old_polygons, new_polygons):
 		print('Processing SVG child elements:', new_element_id)
 
 		overlaps = {}
+		nearest_markers = {}
 
-		polygon = new_polygon
 		for old_element_id, old_polygon in old_polygons.items():
 
+			#FIXME: We don't need the actual intersection, but just if it's too close.
+			#       So if we the bounding boxes aren't close enough to be a problem, then we already don't care.
+			#FIXME: Retrieving bounds is slow, so we should cache them?
 			if not new_polygon.intersects(old_polygon):
-				#FIXME: Get distance to check if someone might have left an accidental gap, or moved too close
-				pass
+				# Get distance to check if someone might have left an accidental gap, or moved too close.
+				#FIXME: This step appears to be slow, so we should make this optional?
+				if False:
+					threshold = stroke_width * 1.5
+					if new_polygon.distance(old_polygon) <= threshold:
+						print("error: Small distance, but not touching")
+						old_point, new_point = shapely_nearest_points(old_polygon, new_polygon)
+						
+						nearest_markers[old_element_id] = shapely.geometry.LineString([old_point, new_point]).buffer(stroke_width)
```

Comment:
This shouldn't have to be `.buffer()`'d. It should just store the 2 points or the `LineString`.
Unfortunately we have no way of dumping non-closed paths right now.